src/uniswapMappings/uniswapPools.ts

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable prefer-const */
+import { Bundle, UniswapPool, Token } from '../../generated/schema'
+import { BigDecimal, BigInt } from '@graphprotocol/graph-ts'
+import {
+  Burn as BurnEvent,
+  Initialize,
+  Mint as MintEvent,
+  Swap as SwapEvent
+} from '../../generated/templates/Pool/Pool'
+import { convertTokenToDecimal } from '../utils'
+import { ZERO_BD } from '../utils/constants'
+import { findEthPerToken, getEthPriceInUSD, sqrtPriceX96ToTokenPrices } from '../utils/pricing'
+
+export function handleInitialize(event: Initialize): void {
+  let pool = UniswapPool.load(event.address.toHexString())
+  pool.tick = BigInt.fromI32(event.params.tick)
+  // update token prices
+  let token0 = Token.load(pool.token0)
+  let token1 = Token.load(pool.token1)
+
+  // update ETH price now that prices could have changed
```

Comment:
ubernit: comments should start with capital letter as you do in other parts

Reply:
some comments are inherited form UniV3 subgraph. Will check that!